deepfence_server/model/scans.go

```diff
@@ -57,8 +57,9 @@ type ScanFilter struct {
 }
 
 type ScanTriggerCommon struct {
-	NodeIds []NodeIdentifier `json:"node_ids" required:"true"`
-	Filters ScanFilter       `json:"filters" required:"true"`
+	NodeIds    []NodeIdentifier `json:"node_ids" required:"true"`
+	Filters    ScanFilter       `json:"filters" required:"true"`
+	IsPriority bool             `json:"is_priority"`
```

Comment:
Let's make this `required:"true"` as well

Reply:
Not sure about making this as required. 
Making this as mandatory will break the existing api clients. 
Also, we are not sure if this needs to be added to the scheduled jobs.